src/components/home/components/overview/OverviewTable.tsx

```diff
@@ -19,6 +19,10 @@ interface Props {
 
 const OverviewTable = (props: Props) => {
     const { columns, rows } = props;
+
+    const gridOptions = {
```

Comment:
Does this work with the API as it is currently?

Reply:
But a change in the API is in pull request which requires future changes